internal/mq/wal/rmq/scanner.go

```diff
@@ -0,0 +1,77 @@
+package rmq
+
+import (
+	"github.com/cockroachdb/errors"
+
+	"github.com/milvus-io/milvus/internal/mq/mqimpl/rocksmq/client"
+	"github.com/milvus-io/milvus/internal/mq/mqimpl/rocksmq/server"
+	"github.com/milvus-io/milvus/internal/streamingnode/server/wal/helper"
+	"github.com/milvus-io/milvus/internal/streamingnode/server/wal/walimpls"
+	"github.com/milvus-io/milvus/internal/util/streamingutil/message"
+)
+
+var _ walimpls.ScannerImpls = (*scannerImpl)(nil)
+
+// newScanner creates a new scanner.
+func newScanner(
+	scannerName string,
+	exclude *rmqID,
+	consumer client.Consumer,
+) *scannerImpl {
+	s := &scannerImpl{
+		ScannerHelper: helper.NewScannerHelper(scannerName),
+		exclude:       exclude,
+		consumer:      consumer,
+		msgChannel:    make(chan message.ImmutableMessage, 1024),
```

Comment:
1024 is too large,  it will consume 5G memory if every message is 5MB.
suggest the length of channel set to 64